### PR TITLE
Added fixer for Standard linter

### PR DESCRIPTION
**Pull request by sumnerevans**
Addresses #780.

@w0rp, is there anywhere that the available fixers are listed?

### CODE REVIEW

File: autoload/ale/fixers/standard.vim [hunk 1]
```diff
@@ -0,0 +1,19 @@
+" Author: Sumner Evans <sumner.evans98@gmail.com>
+" Description: Fixing files with Standard.
+
+function! ale#fixers#standard#GetExecutable(buffer) abort
+    return ale#node#FindExecutable(a:buffer, 'javascript_standard', [
+    \   'node_modules/standard/bin/cmd.js',
```
Comment (w0rp) [inline autoload/ale/fixers/standard.vim]: You should probably ensure that the `cmd.js` version is executed with `node` in the `Fix` function below on Windows. See the eslint function for an example.

**Comment by w0rp** [comment]
@sumnerevans The tools are just listed in the table with the linters. `:ALEFixSuggest` suggests some things from the registry. We don't really need to explicitly mention anywhere that you can also fix files with certain things, it should basically be a given.

**Comment by sumnerevans** [comment]
I'm in the process of adding the Vader tests (I've never done them before so it's a good learning opportunity :smile:). If all goes well, I'll push sometime today or tomorrow.

**Comment by w0rp** [comment]
If you run into trouble installing Docker for running all of the tests, your problem is probably going to be that you need to add your user to a `docker` group and log out and in again. That's the one part of setting up Docker which isn't explained too well.

**Comment by sumnerevans** [comment]
@w0rp, I actually already have Docker installed and I was happy to see how easy it was to run the tests. (I also use Linux which always helps things go smoother :wink:.)

**Comment by w0rp** [comment]
Cool.

**Comment by sumnerevans** [comment]
@w0rp, how does it look now? I added some stuff to the eslint test directory. Should I create a different directory instead? I could also rename the `eslint-test-files` folder to `js-test-files`.

**Comment by w0rp** [comment]
The Fix function looks fine. I don't think you committed the tests, though.

**Comment by sumnerevans** [comment]
@w0rp, whoops. I totally forgot to `git add .`. I updated and force pushed.

**Comment by w0rp** [comment]
Cheers! :beers:

**Pull request merged**